src/app/api/jhipster-mocks/_helpers/repository.ts

```diff
@@ -0,0 +1,21 @@
+import { Prisma } from '@prisma/client';
+import { NextResponse } from 'next/server';
+
+import { unknownErrorResponse } from './api';
+
+export const repositoryErrorResponse = (e: unknown) => {
+  // eslint-disable-next-line sonarjs/no-collapsible-if
```

Comment:
Is it a mistake or not ? as I understand, I think it's for a better readability ?

Reply:
Yeah, it is to keep better readability, I added a comment 👍🏻

Reply:
(will be pushed later)